### PR TITLE
update SuperCollider grid studies

### PR DESCRIPTION
**Pull request by dndrks**
aligns SuperCollider + grid docs with the style of norns studies + syntax. also added `js` highlighting.

the individual `.scd` files have also been updated and re-zipped.

holding until the [`monoM`](https://github.com/dndrks/monom) files have been publicly updated.

### CODE REVIEW

**Comment by dndrks** [comment]
@tehn , these are now ready for review!

File: grid/studies/sc/files/grid-studies-2-1.scd [hunk 1]
```diff
@@ -1,18 +1,16 @@
 (
 Server.default = Server.local;
 
-~m = MonoM.new("/monome", 0);
+~m = MonomeGrid.new("/128", 0);
```
Comment (tehn) [inline grid/studies/sc/files/grid-studies-2-1.scd]: some of these studies using the "128" prefix, others just "monome" ?

**Review by tehn — COMMENTED** [review]
looks fantastic, love the syntax alignment!

File: grid/studies/sc/files/grid-studies-2-1.scd (hunk 1 above)
Comment (dndrks) [reply grid/studies/sc/files/grid-studies-2-1.scd]: woop, great catch! simplified to `"/grid"` with a note in the study text about how that's changeable (within the `"/<string>"` format).

**Review by tehn — APPROVED** [review]

**Pull request merged**